modalstyle.css

```diff
@@ -176,47 +175,54 @@ letter-spacing: 0.03em;
   height: 20px;
   width: 17.985200881958008px;
   left: 0.013671875px;
-  top: 0px;
-  border-radius: 0px;
+  top: 0;
+  border-radius: 0;
   margin-left: 6px;
 }
 
-@media only screen and (max-width: 600px)  {
+.pop-link {
+  text-decoration: none;
+  color: #fff;
+}
 
-  .modal-main{
-    position: absolute; 
-    height: 989px;
-    left: 0px;
-    top: 0px;
-    background: #979493;
-    mix-blend-mode: multiply;
+@media only screen and (max-width: 768px) {
+  #myModal {
+    display: none;
+    position: fixed;
+    left: 0;
+    top: 0;
+    right: 0;
+    width: 100%;
+    height: 100%;
+    padding: 15px 20px;
+    overflow: auto;
+    z-index: 1000;
+    background-color: rgba(208, 208, 209, 0.3);
   }
 
-  .modal-container{
-    height: 85%;
-    width: 335px;
-    top: 60px;
-    border-radius: 0px;
-    margin-left: 8px;
-}
+  .modal-container {
+    width: 100%;
+    border-radius: 0;
+    margin: 0;
+  }
```

Comment:
- [Optional] I see your pop-up is working now @Habtamu210, But my suggestion here the layout is still not aligned well. As you can see it overflows to a side on a `375px screen size`. Please consider fixing this for a better layout. 
![image](https://user-images.githubusercontent.com/69077061/204106004-506cce9c-7827-4f90-8edd-1747513cf809.png)